### PR TITLE
Updates to gRPC with testing on Google Cloud Platform

### PR DESCRIPTION
**Pull request by kibaekkim**
This PR includes the modifications made while testing a gRPC server example on Google Endpoints and Cloud Run.

### CODE REVIEW

**Review by youngdae — COMMENTED** [review]
Looks good to me.

File: src/appfl/run_grpc_server.py [hunk 1]
```diff
@@ -7,40 +9,41 @@
...
 
     # Do not launch a server if it is already on.
-    channel = grpc.insecure_channel(cfg.server.host + ':' + str(cfg.server.port))
-    if grpc_server_on(channel):
-        print("Server is already running . . .")
-        return
+    # channel = grpc.insecure_channel(cfg.server.host + ':' + str(cfg.server.port))
+    # if grpc_server_on(channel):
+    #     print("Server is already running . . .")
+    #     return
```
[the remark this answers is not in the document]
Comment (kibaekkim) [reply src/appfl/run_grpc_server.py:42]: As we discussed, it's not clear yet if we need to check this.

**Comment by kibaekkim** [comment]
@youngdae FYI, https://github.com/APPFL/APPFL/pull/54/commits/59af8004cc8864c03f2dba3fccef5cf5d916f8ef

**Review by minseok-ryu — APPROVED** [review]
Looks great

**Pull request merged**